hack/generator/pkg/astmodel/file_definition.go

```diff
@@ -171,25 +173,27 @@ func createComments(lines ...string) ([]*ast.Comment, int) {
 func (file FileDefinition) SaveToWriter(filename string, dst io.Writer) error {
 	original := file.AsAst()
 
-	// Write generated source into a memory buffer
+	// Write generated source into a memory unformattedBuffer
```

Comment:
minor: Seems weird here to say `unformattedBuffer` when the type is actually `Buffer`?  Maybe phrasing could be improved (or just left how it was?)

Reply:
copy/paste bug?

Reply:
search/replace bug when I renamed the variable. #whoops 😊